velox/functions/prestosql/aggregates/tests/ChecksumAggregateTest.cpp

```diff
@@ -205,14 +205,23 @@ TEST_F(ChecksumAggregateTest, maps) {
 }
 
 TEST_F(ChecksumAggregateTest, rows) {
-  auto row = makeRowVector(
-      {makeFlatVector<int64_t>({1, 3}), makeFlatVector<int64_t>({2, 4})});
+  auto row = makeRowVector({
+      makeFlatVector<int64_t>({1, 3}),
+      makeFlatVector<int64_t>({2, 4}),
+  });
 
   assertChecksum(row, "jMIvLQ5YEVg=");
 
-  row = makeRowVector(
-      {makeNullableFlatVector<int64_t>({1, std::nullopt}),
-       makeNullableFlatVector<int64_t>({std::nullopt, 4})});
+  row->setNull(0, true);
+  assertChecksum(row, "nbYF0I9UTeU=");
+
+  row->setNull(1, true);
+  assertChecksum(row, "DpXXC2Pzbjw=");
+
+  row = makeRowVector({
```

Comment:
nit: can we add a nested row test case ?

Reply:
Added.